src/bg_daemon/fetchers/imgurfetcher.py

```diff
@@ -80,20 +80,21 @@ def __init__(self, filename=None):
         if not filename:
             filename = os.path.join(HOME, "settings.json")
 
-        try:
-            with open(filename) as fp:
-                data = json.load(fp)
-        except Exception as e:
-            raise
+
+        with open(filename) as fp:
```

Comment:
You may want to be opening things with support for universal newlines `'U'` ... very minor.